tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noEmit": true,
+    "skipLibCheck": true,
```

Comment:
This is required now because of typings error in `@types/react-transition-group`